### ISSUE TITLE
Bylaws v2

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.org#13]
From https://github.com/liberapay/salon/issues/32:

> recruit more people into the Liberapay organization, because legally speaking it doesn't exist anymore if it's down to a single member, so it's not safe to have only two members

We could accomplish that by recruiting more codirectors, but we also have the option of modifying the bylaws to add a new category of members who wouldn't have all the powers and responsibilities of the codirectors.

Opinions?

**Comment by Changaco** [comment]
In https://github.com/liberapay/salon/issues/32#issuecomment-215643620 @MartinDelille has volunteered to join the legal org.

**Comment by Changaco** [comment]
*French translation of the first post*

Il faudrait que l'association Liberapay ait plus de membres, car légalement parlant elle serait dissoute de fait si elle n'avait plus qu'un seul membre, donc n'avoir que deux membres n'est pas sûr.

On pourrait recruter des codirecteurs supplémentaires, mais on a aussi l'option de modifier les statuts pour ajouter une nouvelle catégorie de membres qui n'auraient pas tous les pouvoirs et toutes les responsabilités des codirecteurs.

Opinions ?

**Comment by MartinDelille** [comment]
What's the next step to integrate the organization?

**Comment by Changaco** [comment]
I see two possibilities.

If you want to become a codirector then we need to do two things:

1. make sure you understand what you're getting into (article 7 of the bylaws)
2. reach a consensus on your admission (article 2 of the rules of procedure)
3. declare your membership to the administration

If you don't want all the powers and responsibilities of codirectors then we need to:

1. draft the new bylaws
2. adopt them (by consensus, article 2 of the rules of procedure)
3. send the new bylaws to the administration
4. notify all Liberapay users of this change (article 9 of the bylaws)

I recommend reading https://github.com/liberapay/liberapay.org/tree/v1/fr if you haven't already.

One question you should ask yourself is whether you truly agree with Liberapay's neutrality, see #11 for the ugly side of what it means to be neutral.

**Comment by MarionRousseaux** [comment]
Hello Changaco and the other members of the community ! I'm writing you today because I'd like to apply to be part of the college :) Indeed, I'm following Gratipay and Liberapay since a while and I am actually experimenting it right now through different collectives in Lille (Catalyst and Unisson) involved in the commons. We organized an event in Lille on Monday and I had the chance to meet Changaco. It really motivated me to integrate the organization and help you the way I can. Indeed, I'm not a developer but I can maybe explore some subjects with you regarding legal aspects etc. Don't hesitate in asking me more questions, I'll be happy to answer it.

**Comment by Changaco** [comment]
Hey Marion, I'd be happy to have you in the organization. :-)

What you should do is:

1. [Subscribe to the repository](https://github.com/liberapay/liberapay.org/subscription) so you'll receive all the messages (make sure to [enable email notifications](https://github.com/settings/notifications)).
2. The same thing I said to @MartinDelille above: make sure you understand what you're getting yourself into. :D In particular, since the members of the legal org are the ones who can vote to change its bylaws, we need to ensure that we all agree on the principle of neutrality, otherwise we'll have an internal crisis sooner or later. There were some discussions in Lille about Liberapay's neutrality but I'm not sure exactly which ones you were a part of.

I don't know if this is clear or not, feel free to ask questions, you too @MartinDelille.

**Comment by Changaco** [comment]
@Zatalyz Would you be okay with adding @MartinDelille and @MarionRousseaux as codirectors?

**Comment by Zatalyz** [comment]
I'm aggree to adding @MartinDelille and @MarionRousseaux as codirectors. Welcome :)

**Comment by Changaco** [comment]
*French translation of the steps to add a new codirector.*

Pour ajouter un codirecteur il faut :

1. s'assurer qu'il/elle comprend les responsabilités attachées à ce poste (article 7 des statuts)
2. atteindre un consensus sur l'admission (article 2 du règlement intérieur)
3. déclarer le nouveau codirecteur auprès de l'administration française

Le postulant devrait lire https://github.com/liberapay/liberapay.org/tree/v1/fr si ce n'est pas déjà fait.

Le postulant doit s'assurer et confirmer qu'il est d'accord avec le principe de neutralité de Liberapay. Voir #11 pour le côté difficile de la neutralité.

**Comment by MartinDelille** [comment]
Hi! I feel totally fine with the neutrality principle and appreciate the way you handle the ugly case very pragmatically. The only question I have is regarding the eventual risk we carry being codirector. Do you have any idea of thire nature?

**Comment by Changaco** [comment]
I researched the subject a while ago, the keywords are "responsabilité dirigeant association", if you enter them in a search engine you'll find pages likes this one: [La responsabilité des dirigeants - Associations.gouv.fr](http://www.associations.gouv.fr/707-la-responsabilite-des-dirigeants.html).

TL;DR administrators of associations can be prosecuted for various reasons and can be ordered to pay the organization's debts if it goes bankrupt.

**Comment by MartinDelille** [comment]
How could we go bankrupt?

**Comment by Changaco** [comment]
Obviously it's not supposed to happen, but we're a financial service so I can't guarantee that we'll never end up owing money to someone. In fact our first month we ended up owing €0.75 to MangoPay (#9) because we weren't charging users for VAT.

The most obvious way Liberapay can become indebted is through credit card chargebacks (people telling their bank "I didn't want to make that payment, get me my money back"). We try to protect ourselves from that by passing the responsibility to pay back to the users who received the money (terms of service, article 5). We also hold money for a while (currently 4 weeks) to increase the likelihood that the money is still in Liberapay when we receive the order to pay back, but chargebacks can happen much later than 4 weeks after a payment.

TL;DR I'd say the probability of us having to pay Liberapay's debts is very low, and of course I do everything I can to avoid it.

**Comment by MarionRousseaux** [comment]
Hello,

Thank you for welcoming me! :)

I followed the different instructions and read the different
documents/articles etc. I agree with it and with Liberapay's neutrality and
I'll take part in the investigation of legal framework to go deeper on this
aspect. Charly are there more steps to complete? (signature etc.)

Regarding the question of financial risks, I believe that in every
associations there are some. Since Libarapay is a financial service and a
really innovative tool it's hard to know exactly what to expect. As you say
Charly we need to protect ourselves and it could be really important to
keep some money on the side in case of debts or unexpected fees.

Do you think we could organize a crowfunding campaign on Goteo for example?
It would be a good occasion to communicate on Liberapay and ask for a "one
shot" donation in order to have a working capital.
I know it is time-consuming to organize something like that, that's why I
would like to have your opinion on that. I'll actually participate to a
free online training on the 7th of March from 5pm till 6.30pm. Let me know
if you're interested, I'll transfer you the info.

Have a very good evening,




2016-05-27 10:23 GMT+02:00 Charly C. <notifications@github.com>:

> Obviously it's not supposed to happen, but we're a financial service so I
> can't guarantee that we'll never end up owing money to someone. In fact our
> first month we ended up owing €0.75 to MangoPay (#9
> <https://github.com/liberapay/liberapay.org/issues/9>) because we weren't
> charging users for VAT.
>
> The most obvious way Liberapay can become indebted is through credit card
> chargebacks (people telling their bank "I didn't want to make that payment,
> get me my money back"). We try to protect ourselves from that by passing
> the responsibility to pay back to the users who received the money (terms
> of service, article 5). We also hold money for a while (currently 4 weeks)
> to increase the likelihood that the money is still in Liberapay when we
> receive the order to pay back, but chargebacks can happen much later than 4
> weeks after a payment.
>
> TL;DR I'd say the probability of us having to pay Liberapay's debts is
> very low, and of course I do everything I can to avoid it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/liberapay/liberapay.org/issues/13#issuecomment-222089063>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AL3y8MehpKIio1ALstSzdODTrwT64_fBks5qFqnpgaJpZM4IJJLo>
> .
>



-- 


Marion Rousseaux
Linkedin: Marion Rousseaux
+ 33 (7) 82752983

**Comment by Changaco** [comment]
I don't think there's anything you'll need to sign, however I'll need your postal address for the declaration to the administration, which is the last step. You can send that privately by email to admin at liberapay.com or changaco at changaco.oy.lc.

I'm not a fan of the idea of keeping money in the organization's account "just in case". It would be really awkward to have money idling in our bank account while the members of the Liberapay team are severely underpaid, and we would no doubt be tempted to use it in one way or another. We can always ask our community for help as a last resort if and when shit really hits the fan.

*Bonne soirée ;-)*

**Comment by Changaco** [comment]
Oh, I also need your nationality and your "profession" (form: [cerfa 13971*03](https://www.formulaires.modernisation.gouv.fr/gf/cerfa_13971.do)).

**Comment by Changaco** [comment]
Screenshot of the online form:

![spectacle m14488](https://cloud.githubusercontent.com/assets/1581590/15626356/8cc3bd5e-24c3-11e6-92b1-0d3ad802249a.png)

Notice that unlike the paper form the online one has a country field, so we could properly add codirectors who don't live in France. \o/

**Comment by Changaco** [comment]
I've received the info from Marion, I'm waiting for Martin to send his.

**Comment by MartinDelille** [comment]
I just sent it: let's do it :-)

**Comment by Changaco** [comment]
Done.

![spectacle m30810](https://cloud.githubusercontent.com/assets/1581590/15713741/f0979024-2817-11e6-9b0c-d21b3b3f1004.png)

**Comment by Changaco** [comment]
I've just received confirmation that the change has been officially registered. :tada:

**Comment by MarionRousseaux** [comment]
Great !! =)))

2016-06-03 14:34 GMT+02:00 Charly C. <notifications@github.com>:

> I've just received confirmation that the change has been officially
> registered. 🎉
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/liberapay/liberapay.org/issues/13#issuecomment-223567053>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AL3y8CLt4YzU4gwEQnbhW4ktoHda--5mks5qIB9hgaJpZM4IJJLo>
> .
>



-- 


Marion Rousseaux
Linkedin: Marion Rousseaux
+ 33 (7) 82752983

**Comment by MartinDelille** [comment]
I didn't ask but there is no cotisation to be part of the organisation?

**Comment by Changaco** [comment]
There is no membership fee, as stated in the bylaws the association's only source of income is donations.
